packages/shared/ReactTypes.js

```diff
@@ -240,3 +235,78 @@ export type ReactTimeInfo = {
 export type ReactDebugInfo = Array<
   ReactComponentInfo | ReactEnvironmentInfo | ReactAsyncInfo | ReactTimeInfo,
 >;
+
```

Comment:
I assume you're intentionally omitting LegacyHidden because deprecated, and doesn't need shared?

Reply:
Mainly I didn't see it implemented in Fizz originally. I see now that it's just part of the list of pass-throughs that just read children (along with Profiler) but we're not importing it into Fizz so was no reason to.